java/code/src/com/redhat/rhn/domain/scc/SCCCachingFactory.java

```diff
@@ -565,6 +566,33 @@ public static void initNewSystemsToForward() {
                 .getResultList();
     }
 
+    /**
+     * Return list of data for last seen SCC update call
+     * @param cred SCC Org Crednetials
+     * @return a list of maps with the keys scc_login, scc_passwd, checkin
+     */
+    @SuppressWarnings("unchecked")
+    public static List<Map<String, Object>> listUpdateLastSeenCandidates(Credentials cred) {
```

Comment:
Is there any reason not to use `List<SCCUpdateSystemJson>` (other than the name ending with `Json`) here?

Reply:
Well, the Json class is used to serialize the data. Here we have a 1:1 match, but it feels not correct to use the serializer class so deep in the basics